modules/porous_flow/doc/content/source/bcs/PorousFlowEnthalpySink.md

```diff
@@ -0,0 +1,27 @@
+# PorousFlowEnthalpySink
+
+`PorousFlowEnthalpySink` implements a sink that adds heat energy corresponding to
+adding fluid at a mass flux rate (computed by a function) at a specified temperature.
+
+This object should be used in conjunction with (PorousFlowSink)[PorousFlowSink.md]
+that uses the same `flux_function`, so that the correct amount of fluid is injected into the system.
+
+Note that the fluid property object used by this boundary condition should be the same one that is
+used in the computational domain where this object is located.
+
+Parameter [!param](/BCs/PorousFlowEnthalpySink/pressure)
+(along with [!param](/BCs/PorousFlowEnthalpySink/T_in)) is used to calculate
+the injected fluid enthalpy. Most frequently, it is the PorousFlow pressure variable (the porepressure
+in the porous medium). This models the situation where fluid is injected at a specified rate and
+temperature (using this BCs and a PorousFlowSink) which potentially
+leads to changes in the porepressure.
```

Comment:
Needs to be re-written to something like
```
This BC can be used to model two situations.

- If [!param](/BCs/PorousFlowEnthalpySink/fluid_phase) is provided, then the injected enthalpy is calculated using the porepressure within the porous medium, and [!param](/BCs/PorousFlowEnthalpySink/T_in).  This corresponds to injecting at a specified rate and specified temperature which could potentially lead to changes in the porepressure (assuming the appropriate (PorousFlowSink)[PorousFlowSink.md] is also added).
- If [!param](/BCs/PorousFlowEnthalpySink/pressure) is provided, then the injected enthalpy is calculated using this pressure (which may be an AuxVariable, for instance) and [!param](/BCs/PorousFlowEnthalpySink/T_in).  This corresponds to injecting a fluid at a specified rate, specified pressure and specified temperature.

```

Reply:
✅